src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -99,14 +99,24 @@ function runEventInBatch(event) {
 function startWatchingForChangeEventIE8(target, targetID) {
   activeElement = target;
   activeElementID = targetID;
-  activeElement.attachEvent('onchange', manualDispatchChangeEvent);
+
+  if (activeElement.attachEvent) {
+    activeElement.attachEvent('onchange', manualDispatchChangeEvent);
+  } else {
+    activeElement.addEventEventListener('change', manualDispatchChangeEvent, false);
```

Comment:
@jimfb But... 

```js
  doesChangeEventBubble = isEventSupported('change') && (
    !('documentMode' in document) || document.documentMode > 8
  );
```
```js
      if (doesChangeEventBubble) {
        getTargetIDFunc = getTargetIDForChangeEvent;
      } else {
        handleEventFunc = handleEventsForChangeEventIE8;
      }
```

`doesChangeEventBubble` should only be false for all IE8 and thus it shouldn't go down that path for any other browser as far as I can tell. Or am I reading this wrong.